hooks/post/30-upgrade.rb

```diff
@@ -38,6 +38,14 @@ def import_subscriptions
   Kafo::Helpers.execute('foreman-rake katello:upgrades:2.4:import_subscriptions')
 end
 
+def remove_elasticsearch
+  gems = ['ruby193-rubygem-tire', 'tfm-rubygem-tire', 'elasticsearch', 'sigar', 'lucene3']
+  gems.each do |gem|
+    Kafo::Helpers.execute('rpm -e #{gem}')
+  end
+  echo "Remove elastic search files with #rm -rf /var/lib/elasticsearch"
```

Comment:
@jlsherrill - I wasn't sure if your previous comment meant to document during the upgrade, or in the docs themselves. Let me know what you think, thanks!